tailwind.config.js

```diff
@@ -1,61 +1,27 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
+  presets: [
+    require("./src/presets/next.js"),
+    // Note: This is here temporarily until we can figure out how to load the
+    // presets dynamically depending on the template being used.
+    require("./src/presets/classic.js"),
+  ],
   theme: {
```

Comment:
how does it work if `next.js` and `classic.js` have the same color keys but different values? does it override each other or

Reply:
Yeah, the [last configuration will win](https://tailwindcss.com/docs/presets#extending-multiple-presets). In this case we will take the classic one, but if it is configured inside `tailwind.config.js` then it will override that.

So it's like this:

next.js < classic.js < tailwind.config.js

Reply:
oh yikes, yea that's not great since its quite likely that presets use the same keys given that its a common design language and all

for now it seems that somehow we dont have any overlaps but I think quite scary in long term lol